lbrynet/daemon/Publisher.py

```diff
@@ -43,11 +41,13 @@ def create_and_publish_stream(self, name, bid, claim_dict, file_path, claim_addr
         claim_dict['stream']['source']['sourceType'] = 'lbry_sd_hash'
         claim_dict['stream']['source']['contentType'] = get_content_type(file_path)
         claim_dict['stream']['source']['version'] = "_0_0_1"  # need current version here
-        claim_out = yield self.make_claim(name, bid, claim_dict, claim_address, change_address)
+        tx = yield self.wallet.claim_name(
+            name, bid, claim_dict, self.certificate, holding_address
+        )
 
         # check if we have a file already for this claim (if this is a publish update with a new stream)
         old_stream_hashes = yield self.storage.get_old_stream_hashes_for_claim_id(
-            claim_out['claim_id'], self.lbry_file.stream_hash
+            tx.outputs[0].claim_id, self.lbry_file.stream_hash
```

Comment:
always outpoint 0?

Reply:
we create the TX a little bit before this line ourselves, so we know upfront that claim txo is 0.